CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ Version <dev> of the agent adds the ability to filter logs by level, expands ins
 
   Controllers in Rails automatically render views with names that correspond to valid routes. This means that a controller method may not have a corresponding method in the controller class. Code-Level Metrics now report on these methods and don't log false warnings. Thanks to [@jcrisp](https://github.com/jcrisp) for reporting this issue. [PR#2061](https://github.com/newrelic/newrelic-ruby-agent/pull/2061)
 
+- **Bugfix: Private method `clear_tags!` for NewRelic::Agent::Logging::DecoratingFormatter**
```

Comment:
Can you add this to the overview at the top of the Changelog?

Reply:
Oh thank you, i totally blanked on that part. Updated https://github.com/newrelic/newrelic-ruby-agent/pull/2078/commits/2ba74315c09703dde40fff86d1b90e8cbc595ad3